api/src/main/java/com/spotify/ffwd/output/BatchedPluginSink.java

```diff
@@ -37,4 +38,12 @@
      * @return A future that will be resolved when the metrics have been sent.
      */
     public AsyncFuture<Void> sendMetrics(Collection<Metric> metrics);
+
+    /**
+     * Send the given collection of batches.
+     *
+     * @param batches Batches to send.
+     * @return A future that will be resolved when the batches have been sent.
+     */
+    public AsyncFuture<Void> sendBatches(Collection<Batch> batches);
```

Comment:
public keyword is redundant in interface method

Reply:
Yes, but it follows the convention already established in the file.